README.md

```diff
@@ -6,76 +6,86 @@
 [![GitHub release](https://img.shields.io/github/release/kubernetes/dashboard.svg)](https://github.com/kubernetes/dashboard/releases/latest)
 [![Greenkeeper badge](https://badges.greenkeeper.io/kubernetes/dashboard.svg)](https://greenkeeper.io/)
 
-Kubernetes Dashboard is a general purpose, web-based UI for Kubernetes clusters. It allows users to
-manage applications running in the cluster and troubleshoot them, as well as manage the cluster
-itself.
+Kubernetes Dashboard is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications
+running in the cluster and troubleshoot them, as well as manage the cluster itself.
 
 ![Dashboard UI workloads page](docs/dashboard-ui.png)
 
 ## Deployment
-It is likely that the Dashboard is already installed on your cluster. Check with the following command:
-```shell
-$ kubectl get pods --all-namespaces | grep dashboard
-```
 
-If it is missing, you can install the latest stable release by running the following command:
-```shell
-$ kubectl create -f https://git.io/kube-dashboard
+**IMPORTANT:** As of version 1.7 Dashboard uses minimal privileges setup by default. It means, that some manual steps
+are required to make it fully secure and functional. Before performing any steps read [Access Control](
+https://github.com/kubernetes/dashboard/wiki/Access-control) guide.
+
+### Recommended setup
+
+Full security can be ensured only by accessing Dashboard over HTTPS. In order to enable HTTPS mode certificates need
+to be passed to the application. This can be achieved by providing `--tls-cert-file` and `--tls-cert-key` flags to the
+Dashboard. 
+
+Assuming that you have `dashboard.crt` and `dashboard.key` files stored under `$HOME/certs` directory,
+you need to create config map with contents of these files:
+
+```bash
+kubectl apply configmap kubernetes-dasboard-certs --from-file=$HOME/certs -n kube-system
 ```
 
-If you are using Dashboard on a cluster with disabled authorization, you can install the latest stable release by running the following command:
-```shell
-$ kubectl create -f https://git.io/kube-dashboard-no-rbac
+Afterwards, you are ready to deploy Dashboard using following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
 ```
 
-You can also install unstable HEAD builds with the newest features that the team works on by
-following the [development guide](docs/devel/head-releases.md).
+### Alternative setup
 
-Note that for the metrics and graphs to be available you need to
-have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster.
+This setup is not fully secure, however it does not require any additional steps. In this setup access control can be
```

Comment:
This section will be confusing for users. It claims that no additional steps have to be taken. However, from user perspective it is not usable without additional steps (like setup of SSO). I assume this setup version will cause the most effort for user (quite the opposite of the claim). 

Move this page to a seperate wiki page

Reply:
In Deployment section we describe only how to install Dashboard. No additional steps are required to install Dashboard. We can add word 'install' to make it more prominent. First setup requires user to generate certificates and create resource that will be used to mount them into the pod. Here user can simply use single `kubectl create -f...` command.

Why would you move it? IMO README should describe all official ways of deploying Dashboard. There is a secure setup and a bit less secure but can installed quickly.